elevenlabs_mcp/utils.py

```
@@ -0,0 +1,188 @@
+    check = path
+    while not check.exists():
+        parent = check.parent
+        if parent == check:
+            # Reached filesystem root without finding existing dir
+            return False
+        check = parent
+    return os.access(check, os.W_OK)
+
+
+def make_output_file(
+    tool: str, text: str, output_path: Path, extension: str, full_id: bool = False
```

Ruff (0.15.0)

[warning] Unused function argument: `full_id`

(ARG001)